### PR TITLE
574 add support for p11 and remove support for versions 7 8

### LINKED ISSUE #574: Add support for P11 and remove support for versions 7, 8
Current latest stable release of Pharo is P11. Let's have a support for P11, 10, 9 (3 latest versions). Older versions can be removed, so code is less complicated.

### PR DESCRIPTION
**Pull request by Bajger**
- fixes #574

### CODE REVIEW

**Review by glennj — APPROVED** [review]
Nice to be able to remove the old code!

**Comment by Bajger** [comment]
@glennj Thanks for approving! CI is green now for P9-P11.

File: dev/src/BaselineOfExercism/BaselineOfExercism.class.st [hunk 1]
```diff
@@ -55,24 +55,17 @@ BaselineOfExercism >> baseline: spec [
...
-	spec for: #'pharo9.x' do: [ spec package: 'ExercismPharo90' ]
+				group: 'v3' with: #('ExercismV3')
+			].
+			spec for: #'pharo9.x' do: [ 
+				spec package: 'ExercismTools' with: [ spec includes: #('ExercismPharo90') ].
+				spec package: 'ExercismPharo90' ].
+			spec for: #'pharo11.x' do: [
+				"ExercismTools must be loaded prior compatibilty package, to overwrite original class definition of ExTonelWriter"
+				spec package: 'ExercismTools' with: [ spec includes: #('ExercismPharo110') ].
+				spec package: 'ExercismPharo110' with: [ spec requires: 'ExercismTools' ]
```
Comment (glennj) [inline dev/src/BaselineOfExercism/BaselineOfExercism.class.st:66]: This looks like a circular dependency. Is this OK?

Comment (Bajger) [reply dev/src/BaselineOfExercism/BaselineOfExercism.class.st:66]: Actually it is a forward declaration: ExercismTools will work only with ExercismPharo110, but ExercismPharo110 package is dependent on it (ExercismTools must be loaded first). 
See: https://github.com/pharo-open-documentation/pharo-wiki/blob/master/General/Baselines.md#loads-different-packages-depending-on-the-pharo-version

**Review by glennj — APPROVED** [review]

**Comment by Bajger** [comment]
@ErikSchierboom or @kytrinyx Can you please formally approve (from reviewer with write access)? Thanks!

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**